misk-testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -24,6 +25,7 @@ internal class MiskTestExtension : BeforeEachCallback, AfterEachCallback {
         }
         for (module in context.getActionTestModules()) {
           install(module)
+          install(PrometheusHistogramRegistryModule())
```

Comment:
I don't think this is the right place for this. the test should be providing this module. in the future when we don't want to use prometheus histograms, there will be no way to prevent a duplicate binding.

Reply:
`MiskServiceModule` is probably a good place for it